packages/protobuf-bench/README.md

```diff
@@ -10,5 +10,5 @@ server would usually do.
 
 | code generator      | bundle size             | minified               | compressed         |
 |---------------------|------------------------:|-----------------------:|-------------------:|
-| protobuf-es         | 74,150 b      | 36,950 b | 9,642 b |
```

Comment:
This bundle size increased bc of an `esbuild` update on main:  https://github.com/bufbuild/protobuf-es/pull/379